packages/curve-ui-kit/src/shared/ui/ActionInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import { Stack } from '@mui/material'
+import Snackbar from '@mui/material/Snackbar'
+import Alert from '@mui/material/Alert'
+import AlertTitle from '@mui/material/AlertTitle'
+import IconButton from '@mui/material/IconButton'
+import Icon from 'ui/src/Icon'
```

Comment:
We should be using the icons with mui style to be able to use the design system (and not hardcoding the sizes)